processor/servicegraphprocessor/config.go

```diff
@@ -38,7 +38,8 @@ type Config struct {
 	Dimensions []string `mapstructure:"dimensions"`
 
 	// Store contains the config for the in-memory store used to find requests between services by pairing spans.
-	Store StoreConfig `mapstructure:"store"`
+	Store                     StoreConfig `mapstructure:"store"`
+	VirtualNodeFeatureEnabled bool        `mapstructure:"virtual_node_feature_enabled"`
```

Comment:
I'm a bit outdated, but is this how we do feature flags within components? Perhaps @djaglowski knows?

Reply:
Check out the [featuregate](https://github.com/open-telemetry/opentelemetry-collector/tree/main/featuregate) package. It has a good example in the readme, but basically:
1. Use an init func to register a global gate. This returns the value of the gate according to step 2.
2. The gate can only be managed at startup time via use of the `--feature-gates` command line arg. 
3. Save the gate on the component and reference it as necessary.

Reply:
thx, I will polish it with feature flags.